neuro_evolution_ctrnn/brains/CNN_CTRNN.py

```diff
@@ -116,10 +117,10 @@ def _make_convs(cls, config: ConvolutionalNNCfg):
         return conv1, conv2
 
     @classmethod
-    def get_individual_size(cls, config: ConvolutionalNNCfg, input_space: Space, output_space: Space):
+    def get_free_parameter_metadata(cls, config: ConvolutionalNNCfg, input_space: Space, output_space: Space):
         assert output_space == cls.get_output_shape(config, input_space), "output space is not what it should be"
         conv1, conv2 = cls._make_convs(config)
-        return conv1.weight.size().numel() + conv2.weight.size().numel()
+        return {'conv1': conv1.weight.size().numel(), 'conv2': conv2.weight.size().numel()}
```

Comment:
Again i would use a dict to store the strings

Reply:
Although it seems to not be referenced somewhere else and only used for the logging so it may be not necessary here